### PR TITLE
add event APIs

### PR DESCRIPTION
**Pull request by peterc3989**
APIs added: login, signup, subscribeEvent, unsubEvent, createEvent, getUserEvents, getEvents

### CODE REVIEW

File: screens/Login/Login.component.js [hunk 1]
```diff
@@ -7,32 +7,39 @@ import MainButton from '../../components/MainButton/MainButton.component';
...
 
 const Login = ({navigation}) => {
 
 	const [email, setEmail] = useState('');
 	const [password, setPassword] = useState('');
-
+	const { token, setToken} = useContext(UserContext);
 	const login = async () => {
-		const url = 'https://jsonplaceholder.typicode.com/posts';
+		const url = 'http://10.0.2.2:9090/login';
```
Comment (peterc3989) [inline screens/Login/Login.component.js:21]: APIs dont work unless you have the python django backend running locally

Comment (peterc3989) [reply screens/Login/Login.component.js:21]: You need to set up https://github.com/cpp-css/backEndFall2020 and have python,django, and postgres installed
Probably shouldnt merge

Comment (exelarios) [reply screens/Login/Login.component.js:21]: I'll push it once we have the majority of the team backend environment set up.

File: components/EventCard/EventCard.component.js [hunk 2]
```diff
@@ -8,24 +8,77 @@ import { UserContext } from '../../context/UserContext';
...
+		const body = {
+		}
+
+		try {
+            let response = await axios.delete(url, settings);
+            console.log(response.data);
+		} catch (error) {
+			console.error(error);
+		}
+	};
```
Comment (exelarios) [inline components/EventCard/EventCard.component.js]: You might also want to code split this into a separate file, so we don't need to copy the huge chunk of code each time and we can easily just import it.

File: screens/Login/Login.component.js (hunk 1 above)
Comment (exelarios) [reply screens/Login/Login.component.js:21]: We should have an .env file that stores the localhost URL, so whenever the address or port changes we don't need to go in and manually change every one of them.

**Comment by exelarios** [comment]
Merged with #52

**Pull request closed without merging**